### PR TITLE
Add text-spacing feature

### PR DESCRIPTION
**Pull request by ai**
I took data from MSDN.

@Fyrd this PR is important for Autoprefixer 6.3 release https://github.com/postcss/autoprefixer/issues/588

### CODE REVIEW

File: features-json/css-text-spacing.json [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+      "10":"x",
+      "11":"x"
+    },
+    "and_uc":{
+      "9.9":"n"
+    }
+  },
+  "notes":"",
+  "notes_by_num":{
+    "1":"IE accepts -ms-text-autospace property",
```
Comment (cvrebert) [inline features-json/css-text-spacing.json]: Remove the comma here

**Comment by ai** [comment]
@cvrebert fixed. Sorry.

File: features-json/css-text-spacing.json [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+      "33":"n"
+    },
+    "and_chr":{
+      "47":"n"
+    },
+    "and_ff":{
+      "42":"n"
+    },
+    "ie_mob":{
+      "10":"x",
```
Comment (cvrebert) [inline features-json/css-text-spacing.json]: Just "x" without another letter isn't a valid status string

**Comment by ai** [comment]
@cvrebert fixed. Seems like I should not send PR at the morning :).

**Comment by cvrebert** [comment]
Travis is now passing :-)

**Comment by Fyrd** [comment]
Thanks!

**Pull request merged**